docs/content/commands/pack.md

```diff
@@ -72,9 +72,12 @@ paket pack nugets --symbols
 ### Including referenced projects
 
 Paket automatically replaces inter-project dependencies with NuGet dependencies
-if the dependency has it's own [`paket.template`][templatefile]. In addition to
-this the parameter `--include-referenced-projects` instructs Paket to add
-project output to the package for inter-project dependencies that don't have a
+if the dependency has it's own [`paket.template`][templatefile]. Version constraint 
```

Comment:
Please fix spelling:

if the dependency has *its* own... Version constraint*s*

(Not sure if you mean "Version constraint" or "Version constraints, though. In the next line you're implying plural.)